common/ripple/scoreUtils.py

```diff
@@ -16,7 +16,7 @@ async def overwritePreviousScore(userID: int) -> str | None:
 
     # XXX: a bit of a strange dependency -- means the user needs to be online
     # to overwrite a score, but it's not a huge deal. Worth the analytics.
-    user_token = await osuToken.get_token_by_user_id(userID)
+    user_token = await osuToken.get_primary_token_by_user_id(userID)
```

Comment:
Still need to scan through all of these calls, and see if any _should_ be looking for the non-primary tokens.